### PR TITLE
HTS-1097 Corrected get account url

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosaveapi/controllers/HelpToSaveController.scala [hunk 1]
```diff
@@ -104,11 +104,11 @@ class HelpToSaveController @Inject() (helpToSaveApiService:       HelpToSaveApiS
     }
   }
 
-  def getAccount(): Action[AnyContent] = authorised { implicit request ⇒ (authNino, _) ⇒
+  def getAccount(nino: String, systemId: String, correlationId: Option[String]): Action[AnyContent] = authorised { implicit request ⇒ (authNino, _) ⇒
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosaveapi/controllers/HelpToSaveController.scala]: the system ID and correlation ID won't be passed in as parameters to this API through the API platform - we'd read the system ID from config in help-to-save-api and generate a correlationID

Comment (AndyHWChung) [reply app/uk/gov/hmrc/helptosaveapi/controllers/HelpToSaveController.scala]: the NINO wouldn't be passed in either

File: app/uk/gov/hmrc/helptosaveapi/services/HelpToSaveApiService.scala [hunk 2]
```diff
@@ -147,12 +146,14 @@ class HelpToSaveApiServiceImpl @Inject() (helpToSaveConnector: HelpToSaveConnect
...
-                                                             hc: HeaderCarrier,
-                                                             ec: ExecutionContext): GetAccountResponseType = {
+  override def getAccount(nino: String)(implicit request: Request[AnyContent],
+                                        hc: HeaderCarrier,
+                                        ec: ExecutionContext): GetAccountResponseType = {
     validateGetAccountRequest {
       () ⇒
-        helpToSaveConnector.getAccount(nino, correlationId)
+        val correlationId: UUID = UUID.randomUUID()
+        val systemId: String = "mtmp"
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosaveapi/services/HelpToSaveApiService.scala]: it'd be more flexible to read this from config where the default value is `MDTP`

Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosaveapi/services/HelpToSaveApiService.scala]: ok thanks, I have updated my branch to reflect this change.

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**